ledger/src/leader_schedule.rs

```diff
@@ -48,6 +64,28 @@ impl LeaderSchedule {
     pub fn num_slots(&self) -> usize {
         self.slot_leaders.len()
     }
+
+    /// Returns indices starting from 'offset' where pubkey is the leader.
+    pub(crate) fn get_schedule(
+        &self,
+        pubkey: &Pubkey,
+        offset: usize, // Starting index.
+    ) -> impl Iterator<Item = usize> {
+        let index = self.index.get(pubkey).cloned().unwrap_or_default();
+        let num_slots = self.slot_leaders.len();
```

Comment:
nits: "num_slots" -> "epoch_leader_schedule_len`

Reply:
I feel like `num_slots` is more consistent with existing code elsewhere in this struct:
https://github.com/solana-labs/solana/blob/ab0f4c69a/ledger/src/leader_schedule.rs#L48-L50